tests/distributions/test_empirical.py

```diff
@@ -86,3 +86,10 @@ def test_weighted_mean_var(event_shape, dtype):
         with pytest.raises(ValueError):
             empirical_dist.mean
             empirical_dist.variance
+
```

Comment:
You'll need another line here to get the lint test to pass (you could run `make lint` to see if there are any other linting issues).

Reply:
Thanks, oops, thought I'd done that.